### ISSUE TITLE
`packaging` module needed after #6113?

### ISSUE THREAD

**Issue by mih** [datalad/datalad#6312]
```
ERROR: Failure: ModuleNotFoundError (No module named 'packaging')
```

Not sure why it isn't happening all over the place.

**Comment by jwodder** [comment]
@mih At what point is this message appearing?  Note that #6113 added `packaging` to the build-requires in `pyproject.toml`, so datalad now needs to use a PEP 517 backend to install from source.

**Comment by mih** [comment]
It is failing for the `CrippledFS` test run in https://github.com/datalad/datalad/pull/6306

**Comment by jwodder** [comment]
@mih That's not #6113's fault; your PR is based off of master, while #6113 was based off of maint, which has yet to be merged into master.  The failure in #6303 is due to [this line](https://github.com/datalad/datalad/commit/e973d0d82bb172b3f47ae767d4e935b5c2e33ffb#diff-6ee4b4a4d123c4c24e47dfe72d3a3469393a71f7a81b990abf443a379bb88a5aR13) being added by commit e973d0d8 without `packaging` having been added to the requirements.

**Comment by mih** [comment]
Ah, thx for spotting that!

ATM there are extensive conflicts between `maint` and `master`, because of the different fixes to regexes and import reordering. I will put #6306 on hold till I find the time to get this resolved.

**Comment by adswa** [comment]
> ATM there are extensive conflicts between maint and master, because of the different fixes to regexes and import reordering. I will put #6306 on hold till I find the time to get this resolved.

@mih I think @yarikoptic handled these conflicts recently in #6349

**Comment by yarikoptic** [comment]
yeah, I think conflicts are addressed. Let's close, @mih please reopen if there is more todo
